relayer/plugin/relayer.go

```diff
@@ -95,6 +95,22 @@ func NewRelayer(cfg *config.TOMLConfig, lggr logger.Logger, keystore core.Keysto
 
 	// Use config values instead of constants
 	suiClient, err := client.NewPTBClient(
+		loggerInstance,
+		nodeConfig.URL.String(),
+		nil,
+		timeout,
+		keystore,
+		// Use 3 times more concurrency allowance for the main client due to core making
+		// frequent RPC calls to get latest values
+		maxConcurrentRequests*3,
+		client.TransactionRequestType(requestType),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("error in NewRelayer (monitor): %w", err)
+	}
+
+	// Use a separate client for the indexers to avoid rate limiting
+	suiClientIndexers, err := client.NewPTBClient(
```

Comment:
@faisal-chainlink  could you pls expand on this change? How does using seperate client here avoid rateLimit?